samples/nrf9160/slm_shell/README.rst

```diff
@@ -60,7 +63,29 @@ The following table shows how to connect PCA10095 UART_2 to nRF9160 UART_2 for c
      - GPIO GND
 
 .. note::
-   SW9 on nRF9160 DK (VDD IO) should be set to 3V to work with the nRF52 and nRF53 series DK.
+   SW9 on nRF9160 DK (VDD IO) should be set to 3V to work with the nRF53 series DK.
+
+The following table shows how to connect PCA10143 UART_2 to nRF9160 UART_2 for communication through UART:
+
+.. list-table::
+   :align: center
+   :header-rows: 1
+
+   * - nRF7002 DK
+     - nRF9160 DK
+   * - UART TX P1.04
+     - UART RX P0.11
+   * - UART RX P1.05
+     - UART TX P0.10
+   * - GPIO OUT P0.31
+     - GPIO IN P0.31
+   * - GPIO IN P0.30 (optional)
+     - GPIO OUT P0.30 (optional)
+   * - GPIO GND
+     - GPIO GND
+
+.. note::
+   SW9 on nRF9160 DK (VDD IO) should be set to 1.8V to work with the nRF7 series DK.
```

Comment:
```suggestion
   The GPIO output level on the nRF9160 side must be 3 V to work with the nRF7 series DK.
   You can set the VDD voltage with the **VDD IO** switch (**SW9**).
```

Reply:
1.8V, the key point

Reply:
Good catch, I was just copy-pasting to get the formatting and forgot to fix that.